README.md

```diff
@@ -39,6 +39,13 @@ or start on the [landing page](http://localhost:8000/)
 7. Create PR with your changes and attach it for validation on a platform
 
 
+### Commands how to use docker-compose
+- To run containers - docker-compose up -d
```

Comment:
Recommend update documentation to such view
### Commands how to use docker-compose
To run containers 
```bash
docker-compose up -d
```

Reply:
@Alkaponees I've updated file